src/test/java/seedu/address/logic/commands/CountryNoteDeleteCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+
+public class CountryNoteDeleteCommandTest {
```

Comment:
This needs more tests, but can be done in a separate PR

Reply:
I will create a separate issue for this